src/main/scala/io/sdkman/changelogs/DottyMigrations.scala

```diff
@@ -0,0 +1,52 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "026")
+class DottyMigrations {
+
+  @ChangeSet(
+    order = "001",
+    id = "001_add_dotty_candidate",
+    author = "mingchuno"
+  )
+  def migration001(implicit db: MongoDatabase): Candidate = {
+    Candidate(
+      candidate = "dotty",
+      name = "Dotty",
+      description = "Research compiler that will become Scala 3",
```

Comment:
Can we add a bit more detail here please? This is what gets displayed on our website at https://sdkman.io/sdks